.github/workflows/dockerfile_workflow_test.yaml

```diff
@@ -1,6 +1,16 @@
+#
+# This workflow is used to test the `reusable_dockerfile_pipeline` action used
+# to build and push the Docker image to the container registries.
+#
+# The reason this workflow targets the develop branch is so that we can test the
+# action in the PR. If we targeted main, we would need to merge changes into
+# before being able to test them.
```

Comment:
merge change into what?

Reply:
into main

Reply:
TBH I think the grammar here is still incorrect. Proposal to clarify in the comment:

```
If we targeted main, we would need to merge changes into main before being able to test them.
```